packages/angular/src/flyout-menu/flyout-menu.stories.ts

```diff
@@ -0,0 +1,38 @@
+import { boolean, select, withKnobs } from '@storybook/addon-knobs';
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+import {
+  ButtonModule,
+  DialogModule,
+  IconModule,
+  PlaceholderModule,
+} from 'carbon-components-angular';
+
+import { FlyoutMenuModule } from './flyout-menu.module';
+
+storiesOf('Components/Filter menu', module)
+  .addDecorator(
+    moduleMetadata({
+      imports: [ButtonModule, DialogModule, PlaceholderModule, FlyoutMenuModule, IconModule],
+    })
+  )
+  .addDecorator(withKnobs)
+  .add('Basic', () => ({
+    template: `
+      <div style="position: absolute; left: 50%; top: 50%;">
+        <ai-flyout-menu [flip]="flip" [placement]="placement">
+          <div class="title">
+            Filter
+            <a ibmLink (click)="clearFilterClicked($event)" class="clear-flyout" href="#">Clear</a>
+          </div>
+          Columns
```

Comment:
Would it make sense to wrap this in a container `ai-flyout-content` component?

Reply:
What ever you put into the menu becomes its content so for now I don't see the benefit of a separate component.